Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            int number = Lesson3.GetInt(message);
```

Comment:
PromptInt is still a better name, because you not just get an int, you also ask to get it, by printing some message.

Reply:
Please change it all across the program.